desktop/logger/namespaces/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Credit: Modified from `winston-namespace` by @SetaSouto:
+ * 	https://github.com/SetaSouto/winston-namespace
+ */
+
+module.exports = {
+  /**
+   * Boolean indicating if the object is populated with the environment data.
+   */
+  populated: false,
+  /**
+   * Populates the private data 'namespaces' as an array with the environment from the NODE_ENV
+   * environment variable. It splits the data with ',' as separator.
+   * @private
+   */
+  populate: function () {
+    this.namespaces = [process.env.NODE_ENV];
```

Comment:
mmm…

namespaces is supposed to refer to log-namespacing. I think we meant to use another `ENV` here or at least map the `NODE_ENV` to something…

```js
this.namespaces = [process.env.NODE_ENV === 'production' ? '' : '*'];
```

I know I suggested using `NODE_ENV` earlier but I meant to suggest that we use that as a guide to establish the namespace but not use it directly _as_ the namespace, if that makes sense